### DISCUSSION TITLE
Is it possible to use a local container registry?

### DISCUSSION THREAD

**Discussion by patrickdeangelis** [basecamp/kamal#226] [Q&A] [answered]
I want to use mrsk on a VPS, and I don't want to use any external service. Is it possible to use a local container registry?

**Comment by jeremy** [comment, marked as answer]
Sure is. [Configure your MRSK registry](https://github.com/mrsked/mrsk#using-another-registry-than-docker-hub) to use [your own instance](https://docs.docker.com/registry/deploying/) instead of a third-party registry.

(You could even use MRSK deploy to build and boot your registry!)

**Reply by sorenmalling** [reply]
> (You could even use MRSK deploy to build and boot your registry!)

Could you elaborate on this - is it considered a accessory or is it a separate mrsk deploy config?

**Comment by meglio** [comment]
> You could even use MRSK deploy to build and boot your registry!

How am I supposed to do this if it cannot deploy until the registry is available?
